README.md

```diff
@@ -14,13 +14,17 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
 - Company
   - [Charter](docs/company/charter.md)
   - [Roles](docs/company/roles.md)
+  - Marketing
+  - [Pitch](docs/pitch.md)
+  - [Recruitment Ad](docs/recruitment-ad.md)
   - Structure
 - [Communication](docs/communication/communication.md)
 - Leadership
   - [Epic Example](docs/leadership/epic-example.md)
   - [Epic Template](docs/leadership/epic-template.md)
   - [Meetings](docs/leadership/meetings.md)
-- [Teams](docs/teams/teams.md)
+- Learning
+  - [D/HD Workshop Notes](docs/learning/d-hd-workshop.md)
```

Comment:
**question:** What is the difference between the learning and training folder? 🤔 Should we limit this repo to just focus on product/project related stuff rather than Deakin courses? Keen to hear your thoughts to @talzid.

Reply:
I think this is relevant content that helps inform and support the initiatives and work of students so I do think it should be in here. Open to suggestions about the folder it could be in. Can merge Learning and Training